### PR TITLE
W-14231206 run vale on non-tut

### PR DESCRIPTION
**Pull request by hannanelson**

### CODE REVIEW

File: modules/ROOT/pages/int-deploy-mule-apps.adoc [hunk 1]
```diff
@@ -55,7 +55,6 @@ image::int-deploy-output.png["Output panel with Mule Maven Output selected"]
 +
 Your app opens in Runtime Manager in Anypoint Platform.
 +
-Note the domain name for your application.
 The domain name of the application varies, but it follows a similar structure such as: `app-with-100-schedulers.us-w1.cloudhub.io`.
```
Comment (dukesphere) [inline modules/ROOT/pages/int-deploy-mule-apps.adoc]: The image shows `app-with-**40**-schedulers.us-w1.cloudhub.io`

**Review by dukesphere — APPROVED** [review]
approved with one comment to address at your discretion

File: modules/ROOT/pages/int-deploy-mule-apps.adoc (hunk 1 above)
Comment (hannanelson) [inline modules/ROOT/pages/int-deploy-mule-apps.adoc]:
```suggestion
The domain name of the application varies, but it follows a similar structure such as: `app-with-40-schedulers.us-w1.cloudhub.io`.
```

Comment (hannanelson) [reply modules/ROOT/pages/int-deploy-mule-apps.adoc]: Good catch!

**Pull request merged**